backend/models/paypal_webhook_schemas.py

```
@@ -0,0 +1,270 @@
+        payload: Raw webhook payload dictionary
+        
+    Returns:
+        Validated PayPalWebhookPayload object
+        
+    Raises:
+        ValidationError: If payload doesn't match schema
+    """
+    try:
+        return PayPalWebhookPayload(**payload)
+    except Exception as e:
+        logging.error(f"❌ Invalid PayPal webhook payload: {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)